tools/perf/ucc_pt_coll_alltoall.cc

```diff
@@ -57,8 +58,11 @@ void ucc_pt_coll_alltoall::free_coll_args(ucc_coll_args_t &args)
     ucc_mc_free(dst_header);
 }
 
-double ucc_pt_coll_alltoall::get_bus_bw(double time_us)
+float ucc_pt_coll_alltoall::get_bw(float time_ms, int grsize,
+                                   ucc_coll_args_t args)
 {
-    //TODO
-    return 0.0;
+    float N = grsize;
+    float S = N * args.src.info.count * ucc_dt_size(args.src.info.datatype);
+
+    return (S / time_ms) * ((N- 1) / N) / 1000.0;
```

Comment:
Is it really same relation for alltoall?

Reply:
in pairwise alltoall total amount of sent data is np * count * dt_size. N-1 pieces are sent through network and 1 piece just memcopied,  that's why ration is  (N-1)/N